model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb

```diff
@@ -0,0 +1,629 @@
+    "\n",
+    "[The Cancer Imaging Archive (TCIA)](https://www.cancerimagingarchive.net/) is a public service funded by the National Cancer Institute that addresses this challenge by providing hosting and de-identification services to take major burdens of data sharing off researchers. Its rich collection of clinical data and annotations is particularly powerful as a community resource when it is paired with interactive code systems, such as Jupyter systems.\n",
+    "\n",
+    "[itkWidgets](https://github.com/InsightSoftwareConsortium/itkwidgets) allows researchers to visualize images, point sets, and 3D geometry in Jupyter systems (Jupyer Notebooks, JupyterLab, AWS SageMaker, and Google Colab). Despite its name, itkWidgets does not require the use of ITK. It can directly visualize numpy arrays, torch tensors, DASK arrays, VTK polydata, and a multitude of other python data structures."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "nj6P7YCmll4S",
+    "tags": []
+   },
+   "source": [
+    "# Outline\n",
+    "\n",
+    "1. Setup\n",
+    "2. Download data from TCIA\n",
+    "3. Prepare the data with ITK\n",
+    "4. Setup itkWidgets\n",
+    "5. MONAI Zoo Basics\n",
+    "6. MONAI Model Inference\n",
+    "7. Visualizing and Comparing Model and Expert Results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8FgizAMWemJM"
+   },
+   "source": [
+    "# 1. Setup\n",
+    "\n",
+    "These are the initial steps for running notebooks within various Jupyter environments."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "K3DOvHnXsf-I",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Install itk for DICOM I/O and for reading DICOM into an itkImage \n",
+    "#   that manages all DICOM field values, include acquistion details \n",
+    "#   such as voxel image, image orientation, and image directions,\n",
+    "#   which are critical to image processing and display.\n",
+    "\n",
+    "# Upgrade pip, just in case.\n",
+    "!python -m pip install --upgrade -q pip\n",
+    "\n",
+    "# installations required for monai\n",
+    "!python -c \"import monai\" || pip install -q \"monai[nibabel,itk,tqdm,pandas,skimage]\"\n",
+    "\n",
+    "# These are the libraries used to read DICOM Seg objects.\n",
+    "!python -m pip install -q pydicom pydicom-seg\n",
```

Comment:
`pydicom` is included in dev env since monai 0.9.1. Maybe we can move the installation to
```
pip install monai[..., pydicom]
```

Reply:
I defer to you all here.  Does monai also include pydicom-seg or just pydicom?  Could both be moved into the monai install command?

Reply:
I think it's ok to least it as it is for now as monai doesn't currently have a specific requirement for pydicom versions...